operate/webapp/src/main/java/io/camunda/operate/webapp/opensearch/backup/OpensearchBackupRepository.java

```diff
@@ -345,6 +350,10 @@ private List<OpenSearchSnapshotInfo> findSnapshots(
 
   private BackupStateDto getState(
       final List<OpenSearchSnapshotInfo> snapshots, final Integer expectedSnapshotsCount) {
+    final var firstSnapshot = snapshots.get(0);
+    final var startTimeInMilliseconds = firstSnapshot.getStartTimeInMillis();
+    final var lastSnapshot = snapshots.get(snapshots.size() - 1);
+    final var endTimeInMilliseconds = lastSnapshot.getStartTimeInMillis();
```

Comment:
This way we count overall time of backup process starting from beginning of the 1st snapshot and setting timeout for this process. Meaning that if overall backup is running for more than 5 minutes, we will start returning INCOMPLETE state. This is wrong IMO.
Our goal is to catch the time period between two snapshots. If it lasts more than timeout, then we consider the backup INCOMPLETE . I would say that we count `now - endTimeInMilliseconds > timeout` then INCOMPLETE, else IN_PROGRESS.